deployments/scripts/bridge-node-dev-entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
IMHO the file should be `bridge-entrypoint.sh`, since we can run either bridge-node or the relay using it.